bin/check-haproxy.rb

```diff
@@ -80,6 +80,14 @@ class CheckHAProxy < Sensu::Plugin::Check::CLI
          default: 90,
          proc: proc(&:to_i),
          description: 'Session Limit Critical Percent, default: 90'
+  option :backend_session_warn_percent,
+         short: '-b PERCENT',
+         proc: proc(&:to_i),
+         description: 'Per Backend Session Limit Warning Percent, default: 75'
```

Comment:
I realize it doesn't use the default Session Limit Warning, but the description still states this.